src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -7,11 +7,12 @@
 from vivarium_public_health.risks import distributions
 from vivarium_public_health.utilities import EntityString, TargetString
 
+REBIN_UNSUPPORTED = ['unsafe_water_source', 'low_birth_weight_and_short_gestation']
```

Comment:
Why are these unsupported?

Reply:
this was a hold over from the old code. I wasn't sure if there was some conceptual problem with rebinning them? I can try them and see what happens

Reply:
Isn't it the two 2-D risk factors? and I don't think that we have a good idea how to rebin this unless researchers figure out how.

Reply:
so I'm trying with these risk factors now to see if it works (it's taking a while because I don't have caches built for these), but even if I can, should I leave these as unsupported with a note about their being 2D risk factors?

Reply:
Since you made it configurable, they could do this if they wanted.

Reply:
Since you made these configurable, everything should be fine.  The users can rebin risks however they want.

Reply:
Oh, I forgot.  The thing that we want for rebinned risks is to take multiple categories to multiple categories.  e.g.
`rebinned_exposure = {'cat1': ['cat1', 'cat2'], 'cat2': ['cat3'], 'cat3': ['cat4', 'cat5']}`.  This would allow someone to, for instance, collapse one of the dimensions of unsafe water, or to take lbwsg and turn it into a 4-category risk instead of a 50 category one.

This is outside the scope of this ticket, but make sure it's on a ticket somewhere.